### PR TITLE
SAK-34005 - Certification tool - Allow to overwrite unassigned templa…

### PR DESCRIPTION
**Pull request by danielmerino**
…te fields with custom values

### CODE REVIEW

File: tool/src/main/webapp/WEB-INF/jsp/createCertificateFour.jsp [hunk 1]
```diff
@@ -87,13 +87,17 @@
...
                     </tr>
                 </thead>
                 <tbody>
                     <c:forEach items="${certificateToolState.certificateDefinition.fieldValues}" var="tField" >
                         <tr>
                             <td>${tField.key}</td>
                             <td>${tField.value}</td>
+                            <c:if test = "${tField.value == '${unassigned}'}" >
+                            <td><form:input path="templateFields['${tField.key}']" autocomplete="off"/></td>
+        					</c:if>
```
Comment (bjones86) [inline tool/src/main/webapp/WEB-INF/jsp/createCertificateFour.jsp]: Looks like there's some tabs/spaces inconsistency here.

File: tool/src/main/webapp/WEB-INF/jsp/createCertificateThree.jsp [hunk 2]
```diff
@@ -29,11 +29,13 @@
...
-                                            <form:option value="${tField.value}" label="${predefDefault.value}"/>
-                                        </c:if>
-                                    </c:forEach>
+                                	<c:if test="${not empty tField.value}">
+	                                    <c:forEach items="${certificateToolState.templateFieldsToDescriptions}" var="predefDefault" varStatus="index">
+	                                        <c:if test="${tField.key eq predefDefault.key}">
+	                                            <form:option value="${tField.value}" label="${predefDefault.value}"/>
+	                                        </c:if>
+	                                    </c:forEach>
+                                    </c:if>
```
Comment (bjones86) [inline tool/src/main/webapp/WEB-INF/jsp/createCertificateThree.jsp:38]: And in this block.

**Review by bjones86 — CHANGES_REQUESTED** [review]
Functionally this looks good to me. Just a few blocks have some tabs/spaces issues.

I also noticed that the Travis CI failed due to some of the Mock services being out of date (unrelated to the changes here). If you have time to update those while you're in there, that would be great. Otherwise I'll fix them up in a follow up PR.

**Comment by danielmerino** [comment]
@bjones86 thanks for the review, I have fixed the format. Sorry, I haven't never developed Mock services and I'm afraid I will spend a lot of time in doing it...

**Comment by bjones86** [comment]
No problem, @danielmerino. It's really not related to this PR anyway, so I'll submit another PR later to fix them up. Thanks!

**Review by bjones86 — APPROVED** [review]

**Pull request merged**